tests/matrix_free/matrix_vector_hessians_cells.cc

```diff
@@ -185,17 +186,9 @@ test_hessians(const dealii::FE_Poly<dim>                    &fe,
     },
     [&](
       const auto &matrix_free, auto &dst, const auto &src, const auto &range) {
```

Comment:
You can either just remove the parameter names here:
```suggestion
      const auto &, auto &, const auto &, const auto &) {
```
Or add a `[[maybe_unused]]` to all of the parameters:
```suggestion
      [[maybe_unused]] const auto &matrix_free, [[maybe_unused]] auto &dst, [[maybe_unused]] const auto &src, [[maybe_unused]] const auto &range) {
```